packages/p2p/src/components/app.jsx

```diff
@@ -18,6 +18,28 @@ import MyProfile from './my-profile/my-profile.jsx';
 import './app.scss';
 
 const allowed_currency = 'USD';
+
+const TemporaryBarredMessage = () => {
```

Comment:
- Should be an `observer`
- After wrapping it in `observer`, please specify a `displayName`, e.g. 
    - `TemporaryBarredMessage.displayName = 'TemporaryBarredMessage';`